### ISSUE TITLE
Links should be styled as links and match the accessible label

### ISSUE THREAD

**Issue by joshkimux** [AIGAdc/dcdesignweek.org#422]
### Description of issue
Some folks may miss that "this could be you" is a link. Folks who use voice technology to activate links may struggle to access this if they speak the command "Click this could be you" because the accessible label is actually "Email us to become.."

### Expected result
"Email us to become a DCDW sponsor" is styled as a link with the accessible label matching the link's text.

![image](https://netlify-cocoon.netlify.app/.netlify/functions/fetch?code=307&path=eyJzaXRlX2lkIjoiMTRiNDc5ZjMtZWM2YS00ZDBiLWI5NGUtNmZlZmUzMzc4NWUzIiwiZGVwbG95X2lkIjoiNjExZTg5OGE5NDQ4N2IwMDA3NTUxNTAzIiwiaWQiOiI2MWEwODY0Yi0yYjgzLTRlOTgtYTgxYi0xNmM3MjRiY2RhNWYifQ==)

```
Path:      /#skip-to-content
Browser:   Chrome 92.0.4515.131 on Mac OS 10.15.7
Viewport:  1277 x 1009 @2x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-359--dcdesignweek.netlify.app%2F%23skip-to-content&os=OS%20X&os_version=Catalina&browser=Chrome&browser_version=92.0&resolution=1277x1009&speed=1&start=true&ref=netlify-source)

Submitted from [Netlify Deploy Preview #359](https://deploy-preview-359--dcdesignweek.netlify.app/#skip-to-content)

**Comment by marcusrelacion** [comment]
@joshkimux this has been fixed.
